test/evergreen.yml

```diff
@@ -3743,6 +3743,39 @@ buildvariants:
     - name: ".stress-test-4"
     - name: format-abort-recovery-stress-test
 
+- name: ubuntu2004-stress-tests-arm64
+  display_name: Ubuntu 20.04 Stress tests (ARM64)
+  run_on:
+  - ubuntu2004-arm64-large
```

Comment:
We usually use the "small" / "test" distro for testing tasks. I believe it's intentional here to avoid OOM-kill. A comment would be helpful.

Reply:
I added a comment and a WT ticket FIXME to address the issue in the future.